docs/usage/types.md

```diff
@@ -707,3 +708,28 @@ Similar validation could be achieved using [`constr(regex=...)`](#constrained-ty
 formatted with a space, the schema would just include the full pattern and the returned value would be a vanilla string.
 
 See [Schema](schema.md) for more details on how the model's schema is generated.
+
+## Allow Arbitrary Types
+
+You can allow arbitrary types using the `arbitrary_types_allowed` config in the [Model Config](model_config.md).
+
+```py
+{!.tmp_examples/types_arbitrary_allowed.py!}
+```
+_(This script is complete, it should run "as is")_
+
+## Generic Classes as Types
```

Comment:
maybe this is a subsection of "Custom Data Types"?

Reply:
Sure. I'll move the 3 of them under "Custom Data Types" (including `__get_validators__`).